openadapt/models.py

```diff
@@ -127,6 +132,8 @@ class ActionEvent(db.Base):
     screenshot_id = sa.Column(sa.ForeignKey("screenshot.id"))
     window_event_timestamp = sa.Column(ForceFloat)
     window_event_id = sa.Column(sa.ForeignKey("window_event.id"))
+    browser_event_timestamp = sa.Column(ForceFloat)
+    browser_event_id = sa.Column(sa.ForeignKey("browser_event.id"))
```

Comment:
Does the alembic migration script needs to be re-generated? In the script it appears that the foreign key is the timestamp.

Reply:
Done here: c5f24955daed8badca108fdef61d9d91bcf5803c